lib/coxir/model/entities/message/component.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Coxir.Message.Component do
+  @moduledoc """
+  Work in progress.
+  """
+  use Coxir.Model, storable?: false
+
+  @type t :: %Component{}
+
+  embedded_schema do
+    field(:type, :integer)
+    field(:label, :string)
+    field(:style, :integer)
+    field(:custom_id, :string)
+    embeds_many(:components, Component)
```

Comment:
According to [the documentation](https://github.com/discord/discord-api-docs/blob/master/docs/interactions/Message_Components.md#component-object), it looks like we are missing the fields `url`, `disabled` and `emoji`:
```suggestion
    field(:type, :integer)
    field(:style, :integer)
    field(:label, :string)
    field(:custom_id, :string)
    field(:url, :string)
    field(:disabled, :boolean)

    embeds_one(:emoji, Emoji)

    embeds_many(:components, Component)
```